Mage.Sets/src/mage/cards/o/OrcishMine.java

```diff
@@ -0,0 +1,102 @@
+package mage.cards.o;
+
+import java.util.UUID;
+
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.BecomesTappedAttachedTriggeredAbility;
+import mage.abilities.common.BeginningOfUpkeepTriggeredAbility;
+import mage.abilities.common.EntersBattlefieldAbility;
+import mage.abilities.effects.common.DamageAttachedControllerEffect;
+import mage.abilities.effects.common.DestroyAttachedToEffect;
+import mage.abilities.effects.common.SacrificeSourceEffect;
+import mage.abilities.effects.common.counter.AddCountersSourceEffect;
+import mage.abilities.effects.common.counter.RemoveCounterSourceEffect;
+import mage.constants.*;
+import mage.counters.CounterType;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.target.common.TargetLandPermanent;
+import mage.abilities.Ability;
+import mage.abilities.effects.common.AttachEffect;
+import mage.target.TargetPermanent;
+import mage.abilities.keyword.EnchantAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+
+/**
+ *
+ * @author noahg
+ */
+public final class OrcishMine extends CardImpl {
+
+    public OrcishMine(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ENCHANTMENT}, "{1}{R}{R}");
+        
+        this.subtype.add(SubType.AURA);
+
+        // Enchant land
+        TargetPermanent auraTarget = new TargetLandPermanent();
+        this.getSpellAbility().addTarget(auraTarget);
+        this.getSpellAbility().addEffect(new AttachEffect(Outcome.DestroyPermanent));
+        Ability ability = new EnchantAbility(auraTarget.getTargetName());
+        this.addAbility(ability);
+
+        // Orcish Mine enters the battlefield with three ore counters on it.
+        this.addAbility(new EntersBattlefieldAbility(new AddCountersSourceEffect(CounterType.ORE.createInstance(3)), "with three ore counters on it"));
+
+        // At the beginning of your upkeep or whenever enchanted land becomes tapped, remove an ore counter from Orcish Mine.
+        this.addAbility(new BeginningOfUpkeepTriggeredAbility(new RemoveCounterSourceEffect(CounterType.ORE.createInstance()), TargetController.YOU, false));
+        this.addAbility(new BecomesTappedAttachedTriggeredAbility(new RemoveCounterSourceEffect(CounterType.ORE.createInstance()), "enchanted land", false));
```

Comment:
These should be the same triggered ability, which I know is an annoying thing to do. I'm actually wondering if there should be a combined triggered ability class for this sort of thing.

Reply:
Probably, because I copied this from the New Phyrexia shrines, so they're all wrong too. I'll try making a combined triggered ability.

Reply:
Blech, I wish I didn't know about those. If you're willing to make it then be my guest but I don't mind doing it myself.